### ISSUE TITLE
Running Manjaro Sway alongside a different Manjaro edition

### ISSUE THREAD

**Issue by JanisGailis** [Manjaro-Sway/manjaro-sway#174]
First of all, I apologize if this is the wrong place where to ask this! If so, redirect me where appropriate.

I was wondering what's the best way to try out Manjaro Sway as a daily driver, from a different Manjaro edition install. E.g., I'm using Manjaro Gnome, and I'd prefer not to reinstall everything to see if it works for me. I tried installling Sway, and of course that's vanilla Sway without all the goodies that comes with the Manjaro Sway edition out of the box. So I was wondering if there's a way to somehow be able to set everything up in a way that lets me choose between Gnome and Sway before login, and then have a Sway setup similar to the one that comes with this edition.

Thanks!

**Comment by Hessesian** [comment]
Having manjaro i3 version, I tried to edit PKGBUILD per @boredland suggestion, but I received conflict report anyway.

I decided to proceed and replace the i3settings with swaysettins, apparently it didn't break anything.
I'm using lightDM and it didn't manage to start the sway session even though it appeared in selection, so I switched to different tty using ctrl+alt+f2 and logged in and started sway manually.

It loaded the i3 user settings, so I had to copy sway .config files from https://github.com/Manjaro-Sway/desktop-settings/tree/sway/community/sway/etc/skel and add .profile variables to my profile

Afterwords logging in it looks like a fully working sway desktop. Swaylock just keeps locking when I watch youtube videos in Vivaldi, not sure how to go about fixing that one.

Also scratchpad doesn't appear to be working, but overall pretty nice job lads
